bids/grabbids/tests/test_grabbids.py

```diff
@@ -19,6 +19,12 @@ def testlayout2():
     return BIDSLayout(data_dir)
 
 
+@pytest.fixture
+def testlayout3():
```

Comment:
I don't think this is hooked up to anything? The testlayouts are not currently automatically looped over or anything, so we'll either need to implement parameterized tests, or add a couple of tests that test the derivative-specific functionality.